ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -121,24 +121,23 @@ Allele.propTypes = {
   variant: PropTypes.object,
 }
 
-const copyNumberGenotype = (cn, isHemiX) => (
+const copyNumberGenotype = (cn, isHemiX, newline) => (isCalled(cn) && (
```

Comment:
In general, if there is a parameter that is not always set it is best for it to be the last. So since we always explicitly set `newline` but not `isHemiX ` its better to swap the order of these so fo rthe mito cn case it would just be `copyNumberGenotype(genotype.mitoCn, true)`